### ISSUE TITLE
[feature] passive chat

### ISSUE THREAD

**Issue by dai-shi** [dai-shi/remote-faces#40]
The current status message is too limited. We want to chat, but somewhat passively.

Ideas:
- one big chat area instead of one field per user
- a user can send messages many times
- messages will be disappear in a certain period
- [nice to have] reaction to a messaage

We could leave the status message feature, but as people start using "Your Name" as status, so it would be better to have another feature to show a message on an image. So, once this passive chat lands, we will revisit it.
